dbclient/WorkspaceClient.py

```diff
@@ -397,6 +408,8 @@ def log_acl_to_file(self, artifact_type, read_log_filename, write_log_filename,
                     continue
                 acl_resp.pop('http_status_code')
                 write_fp.write(json.dumps(acl_resp) + '\n')
+                write_fp.flush()
```

Comment:
2 questions:
1. I am worried if we do the flushing on every key, the performance can be visibly degraded. Is this something we should worry about?
2. Since the flush will just make sure flushing the python internal buffered data to the os buffered data, are we stating that we are just guarding against the Python app crash scenario, rather than the whole OS system crashing?